app/bundles/EmailBundle/Controller/AjaxController.php

```diff
@@ -228,6 +228,10 @@ protected function testEmailServerConnectionAction(Request $request)
                             $mailer->setHost($settings['amazon_region']);
                         }
                     }
+
+                    if ('mautic.transport.sparkpost' == $transport) {
```

Comment:
You are right, these lines was added in 2.1.0 and removed in 2.1.1, I will remove them for this PR